management/src/clusterctl/main.go

```diff
@@ -161,12 +172,12 @@ func doAction(a actioner) func(*cli.Context) {
 	}
 }
 
-type postCallback func(c *manager.Client, args []string, extraVars string) error
+type postCallback func(c *manager.Client, args []string, flags manager.ActionFlags) error
 type validateCallback func(args []string) error
 
 type postActioner struct {
 	args       []string
-	extraVars  string
+	flags      manager.ActionFlags
```

Comment:
this is more for consistency. `clusterctl` package shall have it's own structure for flags and not need to borrow it from `manager` which is more of a backend package. I think this also falls in line wrt to comments below.